lib/matplotlib/tests/test_mathtext.py

```diff
@@ -433,7 +432,7 @@ def test_mathtext_fallback_invalid():
 @pytest.mark.parametrize(
     "fallback,fontlist",
     [("cm", ['DejaVu Sans', 'mpltest', 'STIXGeneral', 'cmr10', 'STIXGeneral']),
-     ("stix", ['DejaVu Sans', 'mpltest', 'STIXGeneral'])])
+     ("stix", ['DejaVu Sans', 'mpltest', 'STIXGeneral', 'STIXGeneral', 'STIXGeneral'])])
```

Comment:
Why do we need the additional 'STIXGeneral' entries here?

Reply:
I have slightly changed the logic for how the `tspan`s inside a `text` tag are separated (`backend_svg.py::_draw_text_as_text` line 1224 onwards).
During some testing I found poor compatibility with specifying multiple `x` and `y` in the same `tspan` (from reading [the docs](https://www.w3.org/TR/SVG11/text.html#TSpanElement) maybe `dx` would be better if the old way was to be preferred?)
My new solution is cleaner code-wise and seems to resolve the compatibility issues. However, it causes more `tspans` to be generates, hence why the repetition in the tests.